helm/examples/azure/sa/list-sa.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SHARE_NAME=$(cat $VALUES_FILE | grep "^[[:space:]]*shareName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
```

Comment:
As per comments in create-sa.sh , this needs to change/not pickup from a hpcc-azurefile/values.yaml

Reply:
Update the env-sa and this script